src/UI/__tests__/redux.test.js

```diff
@@ -20,16 +33,119 @@ describe('UI reducer', () => {
         isSmallScreen: true,
       });
     });
+
+    describe('auth actions', () => {
+      const errorState = { ...initialState, error: { code: 'auth/error', message: 'Your stuff failed' } };
```

Comment:
The structure here with the states defined above and used in different combos makes it clean and easy to read :)